internal/signalfx-agent/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/signalfx/signalfx-agent
 
-go 1.23.0
```

Comment:
Hum 🤔 will fix this before next round of bumps to see if it avoids the toolchain upgrade issue